nodes/config/tts.html

```diff
@@ -74,7 +37,7 @@
             );
           });
         }
-        $('#node-config-input-voice').val(node.voice);
+        $('#node-config-input-voice').val(node.voice || 'Joanna');
```

Comment:
Instead of having hardcoded string, please use `languages['en-US'].voices[0].id`. We also should move this to `voice` property in `defaults`.

Reply:
To refer to `languages` it needs to be defined before it's used in `defaults` so it means that it should be move to the beginning of the file. Are you fine with this?

Reply:
I would be okay with that if you are fine with it as well.